src/components/components/ebay-notice-base/index.marko

```diff
@@ -15,11 +15,12 @@ static var ignoredAttributes = [
     "mainRoot"
 ];
 
+$ var ariaLabelledBy = input.prefixClass.includes("inline-notice") ? false : "status";
 $ var status = input.status;
 $ var prefixClass = input.prefixClass;
 <${input.root || "section"}
-    aria-labelledby:scoped="status"
-    class=[prefixClass, input.class]
+    aria-labelledby=ariaLabelledBy && component.elId(ariaLabelledBy),
+    class=[prefixClass, input.class],
```

Comment:
Why is there a comma here?

Reply:
the linter told me I needed a comma

Reply:
I think this is a syntax error.
You need to surround the `ariaLabelledBy && component.elId(ariaLabelledBy)` by parentheses.

Reply:
OK I can do that. However I'm pretty sure it actually is legitimate Marko to have the comma there. See 
[Marko issue](https://github.com/marko-js/marko/issues/424)